File: jexler-war/src/main/webapp/WEB-INF/suite/cron.py
from net.jexler import StopSensor
from net.jexler import CronSensor

sensors.add(CronSensor(jexler, "every-minute").setCron("* * * * *").start())
sensors.add(CronSensor(jexler, "every-two-minutes").setCron("* * * * *").start())

while True:
    event = events.take()
    if isinstance(event, CronSensor.Event):
        print "Is is now (jython): %s (%s)" % ("dummy time", event.getSensorId())
    elif isinstance(event, StopSensor.Event):
        break

File: jexler-war/src/main/webapp/WEB-INF/suite/cron.rb
import Java::net.jexler.StopSensor
import Java::net.jexler.CronSensor

$sensors.add CronSensor.new($jexler, "every-minute").setCron("* * * * *").start
$sensors.add CronSensor.new($jexler, "every-two-minutes").setCron("*/2 * * * *").start

begin
  event = $events.take
  if event.is_a? CronSensor::Event
    puts "It is now " + Time.new.inspect + " (" + event.getSensorId + ")"
  elsif event.is_a? StopSensor::Event
    return
  end
end while true
